scripts/blocklist-skylink.sh

```diff
@@ -3,14 +3,49 @@
 set -e # exit on first error
 
 if [ -z "$1" ]; then
-    echo "Please provide a skylink to blocklist" && exit 1
+    echo "Please provide either a skylink or file with skylinks separated by blank lines" && exit 1
 fi
 
+#########################################################
+# read either a file containing skylinks separated by new 
+# lines or a single skylink and put them in an array
+#########################################################
+skylinks=()
+if test -f "$1"; then
+    OLDIFS=$IFS
+    IFS=','
+    while read line
+    do
+        if [[ $line =~ ([a-zA-Z0-9_-]{46}) ]]; then
+            skylinks+=("$BASH_REMATCH")
+        else
+            echo "Incorrect skylink: $line" && exit 1
+        fi
+    done < $1;
+    IFS=$OLDIFS
+else
+    skylinks=("$1") # just single skylink passed as input argument
```

Comment:
no validation on single skylink?

Reply:
Nah, siac will validate the input when you pass it. If it's single skylink then it will just exit with an error and you can fix the skylink. For a list of skylinks it's important to fail early because the list can be 50 skylinks and it would fail somewhere along the way and you would repeat the whole process again.